### PR TITLE
Add snap install instructions

### PR DESCRIPTION
**Pull request by coder3101**
PR for snap installation instruction

### CODE REVIEW

File: content/docs/installation/install-from-binaries/_index.md [hunk 1]
```diff
@@ -23,8 +23,23 @@ Download the `portable.zip` in the assets and unzip it to any directory you like
...
 ### Most Linux Distros
 
+#### AppImage
+
 Download the `AppImage` in the assets, add execute permission to it (`chmod +x <AppImagePath>`), then run it.
 
+#### Snap package
+
+To install latest stable version run the command in the terminal:
+```sh
```
Comment (coder3101) [inline content/docs/installation/install-from-binaries/_index.md:33]:
```suggestion
```sh

```

File: content/docs/installation/install-from-binaries/_index.ru.md [hunk 2]
```diff
@@ -23,8 +23,23 @@ Download the `portable.zip` in the assets and unzip it to any directory you like
...
 ### Most Linux Distros
 
+#### AppImage
+
 Download the `AppImage` in the assets, add execute permission to it (`chmod +x <AppImagePath>`), then run it.
 
+#### Snap package
+
+To install latest stable version run the command in the terminal:
+```sh
```
Comment (coder3101) [inline content/docs/installation/install-from-binaries/_index.ru.md:33]:
```suggestion

```sh
```

File: content/docs/installation/install-from-binaries/_index.zh.md [hunk 3]
```diff
@@ -27,8 +27,23 @@ description: 这是安装 CP Editor 最简单的方式，推荐大多数用户
...
 ### 绝大部分 Linux 发行版
 
+#### AppImage
+
 下载 `AppImage`，执行 `chmod +x <AppImagePath>` 给其添加执行权限，随后运行该文件。
 
+#### Snap package
+
+To install latest stable version run the command in the terminal:
+```sh
```
Comment (coder3101) [inline content/docs/installation/install-from-binaries/_index.zh.md:37]:
```suggestion

```sh
```

File: content/docs/installation/install-from-binaries/_index.md [hunk 4]
```diff
@@ -23,8 +23,24 @@ Download the `portable.zip` in the assets and unzip it to any directory you like
...
 
+#### Snap package
+
+To install latest stable version run the command in the terminal:
+
+```sh
+sudo snap install cpeditor --classic
+```
+
+Bleeding edge alpha version can be installed with
```
Comment (coder3101) [inline content/docs/installation/install-from-binaries/_index.md:38]:
```suggestion
Bleeding edge alpha version can be installed with
```

File: content/docs/installation/install-from-binaries/_index.ru.md [hunk 5]
```diff
@@ -23,8 +23,24 @@ Download the `portable.zip` in the assets and unzip it to any directory you like
...
 
+#### Snap package
+
+To install latest stable version run the command in the terminal:
+
+```sh
+sudo snap install cpeditor --classic
+```
+
+Bleeding edge alpha version can be installed with
```
Comment (coder3101) [inline content/docs/installation/install-from-binaries/_index.ru.md:38]:
```suggestion
Bleeding edge alpha version can be installed with
```

File: content/docs/installation/install-from-binaries/_index.zh.md [hunk 6]
```diff
@@ -27,8 +27,24 @@ description: 这是安装 CP Editor 最简单的方式，推荐大多数用户
...
 
+#### Snap package
+
+To install latest stable version run the command in the terminal:
+
+```sh
+sudo snap install cpeditor --classic
+```
+
+Bleeding edge alpha version can be installed with
```
Comment (coder3101) [inline content/docs/installation/install-from-binaries/_index.zh.md:42]:
```suggestion
Bleeding edge alpha version can be installed with
```

**Comment by coder3101** [comment]
@ouuan or @neko-para please translate and approve.

**Comment by ouuan** [comment]
I think we should wait for a stable snap or release a snap for v6.8.

**Comment by coder3101** [comment]
> I think we should wait for a stable snap or release a snap for v6.8.

Its better we release a v6.8 to stable.

**Comment by ouuan** [comment]
Close this in favor of #249.

**Pull request closed without merging**